plugin-dev/Source/Sentry/Private/HAL/PlatformSentryDefines.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2025 Sentry. All Rights Reserved.
+
+#pragma once
+
+// clang-format off
+
+#ifdef SENTRY_PLATFORM_NULL
+	#define SENTRY_COMPILED_PLATFORM_HEADER(Suffix) \
+		PREPROCESSOR_TO_STRING(PREPROCESSOR_JOIN(Null/Null, Suffix))
```

Comment:
Could we print something like: 

"Sentry supports Unreal Engine on PlayStation, Xbox and Nintendo Switch: https://docs.sentry.io/platforms/unreal/game-consoles/".?